config/tests/jest/setup/setupTests.js

```diff
@@ -1,4 +1,6 @@
 import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
 import { configure } from 'enzyme'
+import 'core-js/stable'
```

Comment:
a te paczki dodałeś do devdependency?

Reply:
w tym PR z SPA ich nie ma w dependency X D * czemu to działa *

Reply:
core js jest ale runtime to dependencja innej paczki, dlatego działa